pkg/runtime/scheme.go

```diff
@@ -302,6 +319,19 @@ func (s *Scheme) Convert(in, out interface{}) error {
 	return s.raw.Convert(in, out)
 }
 
+// Converts the given field label and value for an apiResource field selector from
+// versioned representation to an unversioned one.
+func (s *Scheme) ConvertFieldLabel(version, apiResource, label, value string) (string, string, error) {
+	if s.fieldLabelConversionFuncs[version] == nil {
+		return "", "", fmt.Errorf("No conversion function found for version: %s", version)
```

Comment:
Whoops, I should have caught this, it's not a good default. Should fail by returning its arguments :(